lib/select.ts

```diff
@@ -79,7 +79,7 @@ export function select<T>(
       })
     }],
     selectLabelOnOpen: () => selected.get().label,
-    stretchToContainer: selectBtn
+    stretchToSelector: `.${selectBtn.className.replace(' ', '.')}`
```

Comment:
What about `.${cssSelectBtn.className}`, this being I think more illustrative of how we recommend using it?

Reply:
agree